shacl12-core/index.html

```diff
@@ -5982,6 +5991,52 @@ <h4>sh:in</h4>
 						</div>
 					</aside>
 				</section>
+		
+				<section id="ExpressionConstraintComponent">
+					<h4>sh:expression</h4>
+					<p>
+						Based on <a>node expressions</a>, this section introduces a <a>constraint component</a> called
+						<dfn data-lt="expression constraint">expression constraints</dfn>.
+						Expression constraints can be used in any <a>shape</a> to declare the condition that the
+						<a>node expression</a> specified via <code>sh:expression</code> has <code>true</code> as one of its output nodes.
+						In the evaluation of these node expressions is repeated for all <a>value nodes</a> of the <a>shape</a>
+						as the <a>focus node</a>.
+					</p>
+					<p>
+						<span class="component-class">Constraint Component IRI</span>: <code>sh:ExpressionConstraintComponent</code>
+					</p>
+		
+					<div class="parameters">Parameters:</div>
+					<table class="term-table">
+						<tr>
+							<th>Property</th>
+							<th>Summary and Syntax Rules</th>
+						</tr>
+						<tr>
+							<td><code>sh:expression</code></td>
+							<td>
+								The <a>node expression</a> that must return <code>true</code>.
+								<span data-syntax-rule="expression-scope">The <a>values</a> of <code>sh:expression</code> at a
+								<a>shape</a> must be well-formed <a>node expressions</a>.</span>
+							</td>
+						</tr>
+					</table>
+					<div class="def def-text">
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="Filter">
```

Comment:
```suggestion
						<div class="def-text-body" data-validator="Expression">
```

To fix the definition not being displayed in **Summary of SHACL Core Validators** section